spec/license_checker_spec.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'spec_helper'
+
+describe Repoman::LicenseChecker do
+  let(:repo) { double(Github::Mash, { full_name: 'dummy-repo' }) }
+  let(:checker) { described_class.new(repo) }
+  let(:check_passed) { { result: :passed } }
+  let(:license_text) { '' }
+  let(:license_url) { 'https://raw.githubusercontent.com/dummy-repo/master/LICENSE' }
```

Comment:
Ah, I see. I wonder if there's a way we can make that more explicit? I thought this was testing against a dummy license file…